backend/src/auth_service/privacy_pass.rs

```diff
@@ -9,10 +9,12 @@ use privacypass::{
     private_tokens::{Ristretto255, VoprfServer},
     TruncatedTokenKeyId,
 };
-use sqlx::{Acquire, Postgres, Transaction};
+use sqlx::{Postgres, Transaction};
 use tokio::sync::Mutex;
+use tracing::error;
 
 pub(super) struct AuthServiceBatchedKeyStoreProvider<'a, 'b> {
+    // TODO: Replace with a pool?
```

Comment:
I don't see why not.

Reply:
I removed one reference and stored a connection instead.